bdsx/bds/inventory.ts

```diff
@@ -373,10 +351,10 @@ export class ItemStack extends NativeClass {
         tag.dispose();
         return out;
     }
-    allocateAndSave():CompoundTag {
+    load(tag:CompoundTag|NBT.Compound):void {
```

Comment:
Please don't just move the code without some reason.

Reply:
OK, Sorry for that.
but in this case, the `load` was in `ItemStack`, and I moved it to `ItemStackBase`.
It seems to was happened with the steps of moving the `load` in a later commit.